front/src/components/nft-card-information/NftCardInformation.tsx

```diff
@@ -0,0 +1,33 @@
+import { GCList, GCListItem, GCText } from "../../gc-components";
```

Comment:
pls make sure layout is somehow related to design
you can find template here https://www.figma.com/file/k7cewrEXHk3Ik7Pqftgvff/Untitled?node-id=907%3A3777&t=8hNY2AEqhGCIkAv4-1

Reply:
I didn't know there was a design. Do I need to make the same as here?

Reply:
Nope, I will create separate task for draft design